gammapy/image/maps.py

```diff
@@ -260,7 +260,7 @@ def solid_angle(self):
         """
         Solid angle image
         """
-        xsky, ysky = self.coordinates(mode='edges')
+        xsky, ysky = self.coordinates(coord_type='galactic', mode='edges')
         omega = -np.diff(xsky, axis=1)[1:, :] * np.diff(ysky, axis=0)[:, 1:]
         return Quantity(omega, 'deg2').to('sr')
```

Comment:
`omega` should already be a `Quantity` now, could remove the `Quantity` statement and just return `omega` or `omega.to('sr')` here?